ChangeSet
  1.1603 04/05/04 18:05:45 anton@samba.org +1 -0
  From: John Rose <johnrose@austin.ibm.com>
  
  Please apply the following, which fixes a typo that prevents the
  creation of the manage_flash /proc file.

  arch/ppc64/kernel/rtas_flash.c
    1.14 04/05/04 18:05:12 anton@samba.org +1 -1
    From: John Rose <johnrose@austin.ibm.com>
    
    Please apply the following, which fixes a typo that prevents the
    creation of the manage_flash /proc file.

diff -Nru a/arch/ppc64/kernel/rtas_flash.c b/arch/ppc64/kernel/rtas_flash.c
--- a/arch/ppc64/kernel/rtas_flash.c	Tue May  4 15:40:03 2004
+++ b/arch/ppc64/kernel/rtas_flash.c	Tue May  4 15:40:03 2004
@@ -692,7 +692,7 @@
 	if (rc != 0)
 		goto cleanup;
 
-	manage_pde = create_flash_pde("ppc64/rtas" MANAGE_FLASH_NAME,
+	manage_pde = create_flash_pde("ppc64/rtas/" MANAGE_FLASH_NAME,
 				      &manage_flash_operations);
 	if (manage_pde == NULL) {
 		rc = -ENOMEM;
.........................................................................
# vim: syntax=diff

